### PR TITLE
Feature video feed tests

### PR DESCRIPTION
**Pull request by nicovanbentum**

### CODE REVIEW

File: test/test_video_feed.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+""" Tests the behavior of the VideoFeed class. """
+
+import pytest
```
Comment (BasvRossem) [inline test/test_video_feed.py]: You are not using anything from this import, please remove

File: test/test_video_feed.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    def test_resolution(self):
+        """ Tests if we are able to get and set the resolution correctly. """
+        w, h = 640, 480
+        vf = VideoFeedCV2(0)
+        vf.set_resolution(w, h)
+        res = vf.get_resolution()
+        assert(res[0] == w)
+        assert(res[1] == h)
+
+    def test_open(self):
```
Comment (BasvRossem) [inline test/test_video_feed.py]: I think you should test whether the device is opened before you test your resolution

**Review by BasvRossem — CHANGES_REQUESTED** [review]

File: test/test_video_feed.py (hunk 2 above)
Comment (nicovanbentum) [reply test/test_video_feed.py]: fixed in https://github.com/R2D2-2019/vision/pull/4/commits/bc5c6c49bab12a2013078fc8b5abd4564266563f

File: test/test_video_feed.py (hunk 1 above)
Comment (nicovanbentum) [reply test/test_video_feed.py]: fixed in https://github.com/R2D2-2019/vision/pull/4/commits/bc5c6c49bab12a2013078fc8b5abd4564266563f

**Review by BasvRossem — APPROVED** [review]

**Pull request merged**